### PR TITLE
Lesson 1-6

### PR DESCRIPTION
**Pull request by catbus00**

### CODE REVIEW

File: src/InputWithLabel.js [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+    value,
+    type='text',
+    onInputChange,
+    children
+    }) => {
+
+    const inputRef = useRef();
+    useEffect(() => {
+        inputRef.current.focus();
+    }, []);
```
Comment (dargaCode) [inline src/InputWithLabel.js]: Just one small change here: 

We want the effect to be applied every render, not just the initial render. For that to be the case, we need a fully `undefined` dependency array argument to this `useEffect`, vs the empty array you have currently. With the empty array, it's only triggering on the initial render
```suggestion
    });
```

File: src/AddTodoForm.js [hunk 2]
```diff
@@ -1,33 +1,35 @@
...
         setTodoTitle(newTodoTitle)
     }
     function handleAddTodo(event) {
+
         const newTodo = {
             title: todoTitle,
             id: Date.now()
           };
 
-        event.preventDefault()
```
Comment (dargaCode) [inline src/AddTodoForm.js:16]: try adding this line back in along with the `useEffect` dependency change, and then I believe everything should be working correctly

**Review by dargaCode — CHANGES_REQUESTED** [review]
Looking great! Just a couple small things with the auto-focus. 

I'll leave some notes in the assignment review about the commit message stuff you were asking about =D

File: src/InputWithLabel.js (hunk 1 above)
Comment (catbus00) [reply src/InputWithLabel.js]: Thank you, Darga. Feedback well noted and changes made. Appreciate your guidance.

File: src/AddTodoForm.js (hunk 2 above)
Comment (catbus00) [reply src/AddTodoForm.js:16]: Will do, thank you!

**Review by dargaCode — COMMENTED** [review]
Awesome!

**Pull request merged**